mediasms.go

```diff
@@ -0,0 +1,114 @@
+package mediasms
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/comvex-jp/mediasms-go/errors"
+	"github.com/comvex-jp/mediasms-go/models"
+	"github.com/comvex-jp/mediasms-go/translations"
+)
+
+// Client struct
+type Client struct {
+	Username string
+	Password string
+	Prefix   string
+}
+
+// resultCode from media4u
+type resultCode struct {
+	Result string `json:"result"`
+}
+
+func createSMSID(prefix, messageID string) string {
+	return prefix + messageID
+}
+
+// SMSURL constant for sms-console
+const SMSURL = "https://www.sms-console.jp/"
+
+// Send request to media4u
+func (c Client) Send(messageID string, val models.SendRequest) (models.APIResponse, error) {
+	smsID := createSMSID(c.Prefix, messageID)
+
+	val.SMSID = smsID
+
+	jsonValue, _ := json.Marshal(val)
+	client := &http.Client{}
+
+	req, err := http.NewRequest("POST", SMSURL+"api/", bytes.NewBuffer(jsonValue))
+	req.SetBasicAuth(c.Username, c.Password)
+	req.Header.Set("Content-Type", "application/json")
+
+	resp, err := client.Do(req)
```

Comment:
`err` is defined as a new variable for a second time, isn't this invalid syntax?

Reply:
Apparently not as it threw no errors. I'll use `underscore` since it is not getting checked for anyhow.